easybuild/easyconfigs/n/numdiff/numdiff-5.9.0-GCCcore-10.2.0.eb

```
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'numdiff'
+version = '5.9.0'
+
+homepage = 'https://savannah.nongnu.org/projects/numdiff'
```

Just a very minor thing, but it looks like the "official" homepage of this project is https://www.nongnu.org/numdiff/ (the Savannah page also has a "homepage" link that points to this page)?

```suggestion
homepage = 'https://www.nongnu.org/numdiff/'
```

---

Changed.